.circleci/config.yml

```
@@ -0,0 +1,25 @@
+version: 2.1
+
+jobs:
+  test:
+    docker:
+      - image: circleci/python:<< parameters.python >>
+    working_directory: ~/repo
+    parameters:
+      python:
+        type: string
+      django:
+        type: string
+    steps:
+      - checkout
+      - run: pip install tox
+      - run: tox -e "py$(echo '<< parameters.python >>' | tr -d '.')-django<< parameters.django >>"
```

This is a "complex" command so the title of the step won't be very nice. Maybe you could use that instead.
```suggestion
      - run: 
          name: tox python=<< parameters.python >> django=<< parameters.django >>
          command: tox -e "py$(echo '<< parameters.python >>' | tr -d '.')-django<< parameters.django >>"
```